plane/src/controller/drone.rs

```diff
@@ -140,6 +143,29 @@ pub async fn sweep_loop(db: PlaneDatabase, drone_id: NodeId) {
     }
 }
 
+pub async fn process_pending_actions(
+    db: &PlaneDatabase,
+    socket: &mut TypedSocket<MessageToDrone>,
+    drone_id: &NodeId,
+) -> Result<(), anyhow::Error> {
+    let mut count = 0;
+    for pending_action in db
+        .backend_actions()
+        .pending_actions(drone_id.clone())
+        .await?
+    {
+        let message = MessageToDrone::Action(pending_action);
+        socket.send(message).await?;
```

Comment:
If we attempt to spawn and the backend is already running, it should hit [this condition](https://github.com/drifting-in-space/plane/blob/461da2f4cca6704bcb29dff230c718e9b2503c64/plane/src/drone/mod.rs#L108-L114). Even if that didn't work, the spawn would fail because the container name would clash with the existing backend.

A solution here would be to have `BackendManager::set_state` be a no-op if the state hasn't changed, but I haven't thought through the implications of this and think it's not a big deal right now.